index.html

```diff
@@ -31,6 +32,130 @@ <h1>Marco Almada</h1>
 
+    <section class="projects">
+        <h2>Projects</h2>
+    </section>
+    <section class="box">
+        <h3>Project name goes here</h3>
+        <img class="component1" src="images/component_1.png" alt="website" />
+        <ul class="keywords">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>Javascript</li>
+        </ul>
+        <button class="buttonbox">See this project<img src="images/Union.png"/></button>
+    </section>
+
+    <section class="box2">
+        <h3>Project name goes here</h3>
+        <img class="component2" src="images/component_2.png" alt="website" />
+        <ul class="keywords">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>Javascript</li>
+        </ul>
+        <button class="buttonbox">See this project<img src="images/Union.png"/></button>
+    </section>
+
+    <section class="box3">
+        <h3>Project name goes here</h3>
+        <img class="component3" src="images/component_3.png" alt="website" />
+        <ul class="keywords">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>Javascript</li>
+        </ul>
+        <button class="buttonbox">See this project<img src="images/Union.png"/></button>
+    </section>
+
+    <section class="box4">
+        <h3>Project name goes here</h3>
+        <img class="component4" src="images/component_4.png" alt="website" />
+        <ul class="keywords">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>Javascript</li>
+        </ul>
+        <button class="buttonbox">See this project<img src="images/Union.png"/></button>
+    </section>
+
+    <section class="box5">
+        <h3>Project name goes here</h3>
+        <img class="component5" src="images/component_5.png" alt="website" />
+        <ul class="keywords">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>Javascript</li>
+        </ul>
+        <button class="buttonbox">See this project<img src="images/Union.png"/></button>
+    </section>
+
+    <section class="box6">
+        <h3>Project name goes here</h3>
+        <img class="component6" src="images/component_6.png" alt="website" />
+        <ul class="keywords">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>Javascript</li>
+        </ul>
+        <button class="buttonbox">See this project<img src="images/Union.png"/></button>
+    </section>
+
+    <section class="about">
+        <h1>About</h1>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+            sed do eiusmod tempor incididunt ut labore et dolore magna 
+            aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+             ullamco laboris nisi ut aliquip ea eiusmod
+        </p>
+        <button class="cv">Get my CV  <img src="images/ic_download.svg"/></button>
+
+    </section>
+
+    <section class="languages">
```

Comment:
- [ ] Languages, Frameworks, and Skills are part of the `About` section. So please nest them inside the `About` section.
- [ ] Kindly use flexbox to align the content.